.github/workflows/integration-test-mgr-suite.yaml

```diff
@@ -3,7 +3,6 @@ on: [pull_request]
 
 jobs:
   TestCephMgrSuite:
-    if: ${{ github.event_name == 'pull_request' && github.ref != 'refs/heads/master' }}
```

Comment:
Since this condition is removed, does it mean it will run on all PRs and branches?

Reply:
I believe so with `on: [pull_request]`